conf/up4.bess

```diff
@@ -0,0 +1,517 @@
+# vim: syntax=py
+# -*- mode: python -*-
+# SPDX-License-Identifier: Apache-2.0
+# Copyright(c) 2019 Intel Corporation
+
+from conf.parser import *
+import conf.ports as port
+import conf.sim as sim
+
+port.setup_globals()
+
+
+# ====================================================
+#	Read json config file (START HERE)
+# ====================================================
+
+Uplink = 0x0100
+Downlink = 0x0200
```

Comment:
reason for value change? below its defined as four bytes. Lets put a comment to remind the issue with 1 byte and original value

Reply:
I have removed the direction attribute from the script. It is needlessly causing complications. And it is not really needed over here. We may put it back later in case we discover its necessity.